### PR TITLE
Adminbus-only toxoplasmosis

### PR DESCRIPTION
**Pull request by St0rmC4st3r**
## About The Pull Request

Adds an adminbus disease that turns you into a felinid by infecting your blood with a lot of tiny cats. You need to kill them off if you want to remain unchanged.

## Why It's Good For The Game

Admins can sell bottled tears and scream records of infected players.

## Changelog
:cl:
add: The Syndicate has been studying the new shrinking technology and have managed to turn tiny cats into a horrific disease. But the results were locked away due to uselessness. Right. Right? Right?....
/:cl:

### CODE REVIEW

**Comment by St0rmC4st3r** [comment]
I'll fix it tomorrow.

**Comment by zeskorion** [comment]
image
shouldnt be too hard

**Comment by Cenrus** [comment]
I'll give you bonus points if it makes you speak like a catgirl.

**Comment by St0rmC4st3r** [comment]
> I'll give you bonus points if it makes you speak like a catgirl.

In addition to meowing on random comms? Will try!

**Comment by Cenrus** [comment]
https://github.com/tgstation/tgstation/pull/44567

for maximum cancer

**Comment by zeskorion** [comment]
also- might wanna make some changes to the contagious version, cuz curing doesnt immunize you to transformation diseases

**Comment by St0rmC4st3r** [comment]
@Cenrus, I've seen your message after several hours wasted on OwO talk research and butchering Italian mustache code.

File: strings/owo_talk.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        "cargo":"box",
+        "good":"nya",
+        "sci":"smart kitties",
+        "RD":"smartest cat",
+        "head of":"head kitty of",
+        "nyagger":"best kitten",
+        "catbeast":"kitten",
+        "catbeasts":"kittens",
+        "I":"cat",
+        "me":"kitten"
```
Comment (Cenrus) [inline strings/owo_talk.json]: remove the "me":"kitten" translation or it makes "meow" appear as "kittenow"

Comment (St0rmC4st3r) [reply strings/owo_talk.json]: thanks, done

**Comment by St0rmC4st3r** [comment]
I think this is ready for merging.

**Comment by bloons3** [comment]
Is there a non-contagious version on this?

**Comment by St0rmC4st3r** [comment]
Yes, @bloons3

**Comment by bloons3** [comment]
@St0rmC4st3r 
This no longer appears to infect felinids, is that intentional?

**Pull request merged**